### PR TITLE
Migrate createUpdate.js to styled components

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/3289

# Screenshots

<details>
<summary>Desktop</summary>

image
</details>

<details>
<summary>Mobile</summary>

Screen Shot 2020-11-03 at 10 29 10
</details>

### CODE REVIEW

**Comment by SudharakaP** [comment]
@Betree : The e2e failures seems unrelated to these changes. Are we having any known e2e test suite issues? 🤔

**Comment by Betree** [comment]
Yes `39-recurring-contributions.test.js` is pretty unstable, we need to work on it or disable it. Feel free to restart the tests when you're not sure.

File: pages/createUpdate.js [hunk 1]
```diff
@@ -91,13 +91,6 @@ class CreateUpdatePage extends React.Component {
 
     return (
       <div className="CreateUpdatePage">
```
Comment (Betree) [inline pages/createUpdate.js:93]:
```suggestion
      <div>
```

Comment (SudharakaP) [reply pages/createUpdate.js]: Done. 👍🏽

**Pull request merged**